js/foam/u2/DAOListView.js

```diff
@@ -90,9 +91,9 @@ CLASS({
           return;
         }
 
-        var child = obj.toE ?
-            obj.toE(this.Y) :
-            obj.toRowE ? obj.toRowE(this.Y) :
+        var child = obj.toRowE ?
+            obj.toRowE(this.Y.sub({ data: obj })) :
+            obj.toE ? obj.toE(this.Y.sub({ data: obj })) :
             this.rowView({ data: obj });
```

Comment:
Rewired this. `rowView` now defaults to empty, and has highest precedence (since that allows for overriding from attributes and so on). The `toRowE`, then `toE`, and finally falling back to a default `DetailView`.